test-cases/gemini/gemini-3h-cdc-preimage-write.yaml

```diff
@@ -0,0 +1,24 @@
+test_duration: 300
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+instance_type_db: 'i3.large'
+
+store_results_in_elasticsearch: False
+user_prefix: "gemini-cdc-preimage-write"
```

Comment:
preimage isn't enabled in gemini_table_options

Reply:
Fixed